src/regions/ca/info/ca-qc/reporting.fr.md

```diff
@@ -1,16 +1,14 @@
-## Québec: Qui dois-je contacter si je pense avoir le COVID-19 ?
-
-Le 13 mars 2020, l’état d’urgence sanitaire a été déclaré sur tout le territoire québécois. Cette mesure exceptionnelle octroie au gouvernement du Québec les pouvoirs nécessaires pour mettre en place l’ensemble des mesures visant à assurer la protection de la santé de la population.
+## Guide autosoins - COVID-19
 
-Une attention particulière doit être portée aux personnes âgées, particulièrement celles de 70 ans et plus, puisqu’elles sont davantage à risque de complications.
+Le [Guide autosoins](https://publications.msss.gouv.qc.ca/msss/document-002491/) a pour but d'aider chaque personne à prendre les meilleures décisions possible pour sa santé et celle de ses proches durant la pandémie de COVID-19. Il permet plus spécifiquement de connaître les meilleurs moyens de se protéger de la COVID-19, de prendre soin de soi ou de donner les soins de base à son entourage et de savoir où et quand consulter si l'on a besoin de soins de services.
 
-Si la COVID‑19 vous inquiète ou si vous présentez des symptômes comme de la toux ou de la fièvre, vous pouvez contacter, sans frais, le 418 644-4545 pour la région de Québec, le 514 644-4545 pour la région de Montréal, le 450 644-4545 pour la région de Montérégie, le 819 644-4545 pour la région l’Outaouais ou le 1 877 644-4545 ailleurs au Québec.
+## Québec: Qui dois-je contacter si je pense avoir le COVID-19 ?
 
-De plus, si vous avez de la toux ou de la fièvre :
+De plus, si la COVID‑19 vous inquiète ou que vous avez de la toux ou de la fièvre :
```

Comment:
I'd remove `de plus` here since it is the first sentence of the paragraph

Reply:
Fixed!